pages/blog/2023-summary.md

```diff
@@ -0,0 +1,169 @@
+
+<Figure
+  src="/img/posts/2023-summary/Top-5-members.webp"
+/>
+
+
+Additionally, our weekly activity continues to grow as we run different mentorship programs and initiatives that bring traction to our community.
+
+<Figure
+  src="/img/posts/2023-summary/slack-activity.webp"
+/>
+
+
+### Social Media 
+
+Social media is a medium we use for sharing news and updates around our community. We don't use it for marketing purposes but mainly for communication.
+
+
+> And honestly, I didn't invest much when it came to socials, so don't expect big big numbers 😝
+
+Let's dive into the data:
+
+#### LinkedIn
+
+For our LinkedIn page, we are currently sitting at **2988** overall followers, and during 2023, we gained a total of **849** followers since 2022.
+
+<Figure
+  src="/img/posts/2023-summary/linkedin.webp"
+/>
+
+We also don't use any premium or marketing services; everything you see here is 100% organic, and even though LinkedIn still has much to offer users, it is way better than the X/Twitter app, in my opinion.
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-LinkedIn.webp"
+/>
+
+#### X/Twitter
+
+I don't have much to say regarding Twitter, but on the bright side, we gained almost 1K followers since 2022, and we currently have 4.4k overall followers. When it comes to website referrals via Twitter, we have seen a decline compared to last year, 2022
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-Twitter.webp"
+/> 
+
+Although the decline is shallow, I'm not extremely worried. Most top tweets for this year were appreciation tweets balanced up with technical tweets. Additionally, we don't use any paid services, and having any additional investment in this platform is not worth it, at least for now.
+
+#### YouTube
+
+The AsyncAPI YouTube channel performed way better than expected, especially as we moved from online to in-person conferences. Our live streams and conference recordings bring in a lot of views, and I'm optimistic about 2024 as well.
+
+<Figure
+  src="/img/posts/2023-summary/youtube.webp"
+/> 
+
+#### Newsletter
+On the 5th of April, 2023, we sent out our first AsyncAPI Community Updates Newsletter. The main aim was to summarize monthly activities and initiatives in the community straight to your inbox.
+
+<Figure
+  src="/img/posts/2023-summary/success-sent.webp"
+/> 
+
+Out of over 1.6k successful emails sent over the last nine months, the December Edition was the most popular. The reason is that that was when we released the AsyncAPI v3, one of our huge accomplishments in 2023.
+
+<Figure
+  src="/img/posts/2023-summary/total-opens.webp"
+/> 
+
+And, of course, I'm not exempt from unsubscribers. Although the number might seem minimal to some, it still stings as a writer.
+
+<Figure
+  src="/img/posts/2023-summary/unsubscribed.webp"
```

Comment:
don't worry, it is better to see people unsubscribe than having a huge number of subscribers and not many people opening emails

Reply:
thats true